src/foam/util/Emails/EmailsUtility.java

```diff
@@ -57,6 +59,25 @@ public static void sendEmailFromTemplate(X x, User user, EmailMessage emailMessa
       }
     }
 
+    SupportConfig supportConfig = theme.getSupportConfig();
+
+    // Set ReplyTo, From, DisplayName in emailMessage 
+    EmailConfig emailConfig = supportConfig.getEmailConfig();
+    // REPLY TO:
```

Comment:
If emailConfig is null or emailConfig.get... are null, then need to use the emailConfig from the context.
so perhaps
EmailConfig emailConfig = (EmailConfig) x.get("emailConfig").fclone();
EmailConfig supportEmailConfig = supportConfig.getEmailConfig();
emailConfig.copyFrom(supportEmailConfig);
then use emailConfig ...